### DISCUSSION TITLE
Should we move uBlock source code here?

### DISCUSSION THREAD

**Discussion by lambdakilo** [uBlockOrigin/uAssets#20096] [Q&A]
Should we move https://github.com/gorhill/uBlock under this organization?

**Comment by MasterKia** [comment]
https://github.com/gorhill/uBlock/issues/3726#issue-330818746

**Reply by lambdakilo** [reply]
Thanks for the reply. I've seen that but I was thinking if that is really the most optimal way to handle this.
